internal/pkgecosystem/npm.go

```diff
@@ -12,7 +15,7 @@ type npmJSON struct {
 	} `json:"dist-tags"`
 }
 
-func getNPMLatest(pkg string) (string, error) {
+func getNpmLatest(pkg string) (string, error) {
```

Comment:
NPM is the correct capitalisation per https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
Ah, thanks for pointing that out